src/blockchain.js

```diff
@@ -59,19 +65,29 @@ class Transaction{
 }
 
 class Block {
+    /**
+     * @param {Date} timestamp 
+     * @param {Transaction[]} transactions 
+     * @param {string} previousHash 
+     */
     constructor(timestamp, transactions, previousHash = '') {
         this.previousHash = previousHash;
-        this.timestamp = timestamp;
+        this.timestamp = timestamp.toISOString();
         this.transactions = transactions;
         this.nonce = 0;
         this.hash = this.calculateHash();
-        
     }
 
+    /**
+     * @returns {CryptoJS.WordArray}
```

Comment:
This method actually returns a string. Not a CryptoJS object.

Reply:
I Checked this, (hover over) and you see it returns a CryptoJS.WordArray, but yes, it can be a normal string

Reply:
Don't trust Visual Studio Code's Intellisense.
CryptoJS isn't even used to generate wallet address, so I have no idea how it thinks this is acceptable!

Reply:
oh, wait, i missed the .toString ... Sorry, i will chnage that ...